docs/contributing/how-to-write-a-reference-guide.md

```diff
@@ -0,0 +1,175 @@
+---
+title: How to Write a Reference Guide
+---
+
+## What are reference guide articles?
+
+Reference guide articles cover discrete topics as documentation with links to other resources. A reference guide explains a Gatsby concept or technique without the step-by-step context provided by a tutorial or recipe.
+
+Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios. Reference material should list text headings for each section of a guide, rather than "steps 1-5" like a tutorial.
+
```

Comment:
```suggestion

## What are reference guide overview pages?

**Reference guide overview pages** act as parent pages for groups of reference guide articles, offering an original summary of what can be found in the section. They almost always include a `<Guidelist />` component to automatically list the subpage links in that section of the sidebar. See the [Docs and Blog Components](/contributing/docs-and-blog-components/#guide-list) page to learn more about the Guidelist component.
```

Since this doc now covers both things, we should mention both formats up front.